Makefile

```diff
@@ -398,6 +400,10 @@ check-capi:
 build-wasmer:
 	$(CARGO_BINARY) build $(CARGO_TARGET_FLAG) --release --manifest-path lib/cli/Cargo.toml $(compiler_features) --features="webc_runner" --bin wasmer
 
+
+build-wasmer-debug:
```

Comment:
This is already present in the Makefile, same command but with the added "debug" feature.

Reply:
Ah, yeah, I missed that.

Reply:
I've reverted the changes so `Makefile` matches `master` again.